docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| CA/Browser Forum Onion Extension  | MAY          | N            | __**TODO**__ |
+| Signed Certificate Timestamp List | MAY          | N            | See [Section 7.1.2.9.4](#71294-signed-certificate-timestamp-list) |
+| `subjectKeyIdentifier`            | SHOULD NOT   | N            | See [Section 7.1.2.9.5](#71295-subject-key-identifier) |
+| Any other extension               | SHOULD NOT   | -            | __**TBD**__ |
+
+##### 7.1.2.6.7 Authority Information Access
+
+The `AuthorityInformationAccesssSyntax` MUST contain one or more `AccessDescription`s. Each `AccessDescription` MUST only contain a permitted `accessMethod`, as detailed below, and each `AuthorityInformationAccessSyntax` MUST contain all required `AccessDescription`s.
+
+| __Access Method__ | __OID__            | __Presence__ | __Access Location__         | __Description__ |
+| --                | --                 | -            | ----                        | ---             |
+| `id-ad-ocsp`      | 1.3.6.1.5.5.7.48.1 | MUST         | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's OCSP responder. |
+| `id-ad-caIssuers` | 1.3.6.1.5.5.7.48.2 | SHOULD       | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's certificate. |
+| Any other value   | -                  | MUST NOT     | -                           | -               |
+
+##### 7.1.2.6.8 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be FALSE |
+| `pathLenConstraint` | MUST NOT be present |
+
+**Note**: CAs MUST observe DER encoding rules, such as not explicitly encoding DEFAULT values within OPTIONAL fields.
+
+##### 7.1.2.6.9 Certificate Policies
+
+| __Field__                  | __Presence__ | __Description__ |
+| ---                        | -            | ------          |
+| `certificatePolicies`      |              |                 |
+| \ \ **1**                  | MUST         | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` |              | The Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)) associated with the given Subscriber Certificate type (see [Section 7.1.2.6.1](#71261-subscriber-certificate-types). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **2**                  | MAY          | The Issuing CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the Issuing CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **3**                  | MUST NOT     | The Issuing CA MUST NOT include any additional `PolicyInformation` values that do not meet the above profile. |
```

Comment:
No, it's not possible that more policyIdentifiers meet this profile. Any additional policy identifiers meet a **different** profile. That profile may be a subset, or may incorporate by reference these BRs, but that's a different profile, and a different trust framework. These are problematic, and our goal is to prohibit them in a follow-up ballot, but for now, they are grandfathered in under **2**.

I **think** what you're highlighting is instead confusion about the syntax for how to represent items in a set. The intent is that **2** can contain multiple identifiers (e.g. if X, Y, Z all meet the requirements of **2**, then you can include X, Y, Z).  This is what I called out in the first bullet, in my email https://archive.cabforum.org/pipermail/validation/2021-March/001645.html

What we're saying here is you MUST NOT include any content that does not meet the requirements of 1 or 2. And you MUST include 1 as the first item, since the ordering actually matters to several clients (Chrome and Firefox as two examples).

I'll leave this open, because I think this discussion is about presentation and trying to find a way to unambiguously represent that.

Reply:
I read 2 as allowed to include just one policyIdentifier. Thanks for clarifying.